db/migrate/20250307153923_migrate_application_type_with_local_authority_id_data.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class MigrateApplicationTypeWithLocalAuthorityIdData < ActiveRecord::Migration[7.2]
+  class ApplicationType < ActiveRecord::Base; end
+  class LocalAuthority < ActiveRecord::Base; end
+
+  def change
+    up_only do
+      ApplicationType.all.find_each do |application_type|
```

Comment:
```suggestion
      ApplicationType.where(local_authority_id: nil).find_each do |application_type|
```

To account for the case where this has been rolled back and reapplied, or if application types have been created manually.

Reply:
As we are doing it in same PR it should be any case of this. Although happy to take this into account

Reply:
Yeah, I'm mainly thinking about for dev instances where it's not really vital but might be annoying to fix :)